libraries/AP_Airspeed/AP_Airspeed.h

```diff
@@ -7,6 +7,14 @@
 #include <AP_Math/AP_Math.h>
 #include <AP_MSP/msp.h>
 
+
+#ifndef HAL_AIRSPEED_ENABLED
+// Build Airspeed for only Plane, Sub, & Rover and SITL
+#define HAL_AIRSPEED_ENABLED APM_BUILD_TYPE(APM_BUILD_ArduPlane) || APM_BUILD_TYPE(APM_BUILD_Rover)  || \
+                             APM_BUILD_TYPE(APM_BUILD_ArduSub)   || APM_BUILD_TYPE(APM_BUILD_Replay)
```

Comment:
I need to add the following in the define or something else in order to ensure the status flag in `GCS_MAVLink\GCS.cpp` get complied in?

```suggestion
                             APM_BUILD_TYPE(APM_BUILD_ArduSub)   || APM_BUILD_TYPE(APM_BUILD_Replay) \
                             APM_BUILD_TYPE(APM_BUILD_DIRECTORY)                             
```